source/query/filter.txt

```diff
@@ -364,6 +364,35 @@ The query returns the following document:
       "dateCreated": { "$date": "2003-03-26" }
    }
 
+Match by Nested Field
+~~~~~~~~~~~~~~~~~~~~~
+
+This example performs the following operations:
+
+- Adds a document with ``first`` and ``last`` subfields nested in the
+  ``name`` field
+- Finds the document with the ``last`` subfield of "Washington"
+
+Add the following document to your collection:
+
+.. code-block:: shell
```

Comment:
Can we use "javascript" highlighting here so we get the field name highlights?

Reply:
Still have this question from last round of review.

Reply:
Actually ignore this! As discussed on Slack we can leave "shell" here to be consistent with the rest of hte page.